card/mod/05_standard/set/self/signin.rb

```diff
@@ -14,34 +14,34 @@ def default_title_args args
     # annoying step designed to avoid table of contents.  sigh
     _render_core( args )
   end
-  
+
   view :closed_content do |args|
     ''
   end
-  
+
   def default_core_args args={}
     args[:buttons] = button_tag 'Sign in', :situation=>'primary'
     if Card.new(:type_id=>Card::SignupID).ok? :create
       args[:buttons] += link_to( '...or sign up!', card_path("account/signup"))
     end
-    args[:buttons] += raw("<div style='float:right'>#{ view_link 'RESET PASSWORD', :edit }</div>") #FIXME - hardcoded styling
+    args[:buttons] += raw("<div style='float:right'>#{ view_link 'RESET PASSWORD', :edit, :path_opts=>{:slot=>{:hide=>:edit_toolbar}} }</div>") #FIXME - hardcoded styling
     args
```

Comment:
> I would think the toolbar would only come with the views with frames
That's correct.
> We have to start sticking this on other views now? 
No. The edit view is framed but it doesn't make sense to show the toolbar for the signin card.